### PR TITLE
Task/common94 finalize auth

### PR DESCRIPTION
**Pull request by foxt451**
This branch finalizes the auth flow without Google auth, i.e.: account verification and password restoration, including improvements and bug fixes from previous branches. This branch incorporates Barracuda713's task for frontend validation, also his task for adding confirm password input, and my previous task for backend validation improvement.

### CODE REVIEW

**Comment by foxt451** [comment]
[Finalize authorization flow](https://trello.com/c/0x0tjKTN/94-finalize-authorization-flow)

**Comment by foxt451** [comment]
[Add separate secrets for Access and Refresh tokens](https://trello.com/c/fhtFVhXF/75-add-separate-secrets-for-access-and-refresh-tokens)

File: frontend/src/pages/account-verification-page/account-verification-confirm-page.tsx [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        setIsLoading(true);
+        // remember that token in url is encoded to base64url
+        const response = await responsePromise;
+        navigate(AppRoutes.SIGN_IN, { replace: true });
+        setNotification({
+          type: NotificationType.SUCCESS,
+          iconName: IconName.BELL,
+          title: 'Account Verification',
+          message: response.message,
+          durationMs: 10_000,
```
Comment (Barracuda713) [inline frontend/src/pages/account-verification-page/account-verification-confirm-page.tsx]: Move it and other notification objects to auth/config/config.ts

**Review by Barracuda713 — CHANGES_REQUESTED** [review]

**Review by Barracuda713 — DISMISSED** [review]

**Review by Barracuda713 — APPROVED** [review]

**Comment by foxt451** [comment]
[Validation on back-end side for Sign-in / Sign-up + restore password](https://trello.com/c/pGBuAZ4a/61-validation-on-back-end-side-for-sign-in-sign-up-restore-password)

**Review by kyrylo-burmelov — APPROVED** [review]

**Pull request merged**